kolibri/plugins/management/assets/src/device_management/state/mutations/deviceInfoMutations.js

```diff
@@ -0,0 +1,3 @@
+export function SET_DEVICE_INFO_PAGE_STATE(state, pageState) {
+  state.pageState = pageState;
```

Comment:
I still dislike this kind of mutation.

Reply:
Plus it's identical to all the other page state mutations in the module!

Reply:
yeah .. was just being consistent with the rest of the module